backend/api_server/app.py

```diff
@@ -23,6 +25,39 @@
 configure_logging(APP_NAME)
 
 
+def has_datadog_env_vars():
+    return os.environ.get("DD_AGENT_HOST", None) and os.environ.get("DD_TRACE_AGENT_PORT", None)
+
+
+if has_datadog_env_vars():
+    # Datadog APM tracing
+    # See https://ddtrace.readthedocs.io/en/stable/basic_usage.html#patch-all
+
+    # TODO: Fix this! next line may be redundant with DD_GEVENT_PATCH_ALL env var
+    # in .happy/terraform/modules/service/main.tf
+    # see ticket: https://github.com/chanzuckerberg/single-cell-data-portal/issues/5821
+    # gevent.monkey.patch_all()
```

Comment:
Is this the redundant line or `tracer.configure`?

Reply:
Thanks for catching this! I will remove it. Oversight on my part